src/components/shared/snackNotification.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div id="ConnectionStatusSnack-ctn">
+    <v-snackbar v-model="visibility">
+      <span id="snack-message">
+        {{ message }}
+      </span>
+
+      <template #action="{ attrs }">
+        <div
+          v-bind="attrs"
+          id="close-icon"
+          @click="handleClose"
```

Comment:
@vickywane what about the case when the snackbar closes itself due to timeout and not a click?

Reply:
I don't understand what you are trying to point out. 

The `x` icon for closing the snackbar manually, and it can also close after a timeout

Reply:
The point is that visibility is only switched to false on click, but not if the snackbar closes due to timeout. In effect the problem with showing the latest old message when switching between menu items sill remains.